api_tests/providers/preprints/serializers/test_serializers.py

```diff
@@ -1,12 +1,59 @@
 # -*- coding: utf-8 -*-
 from nose.tools import *  # flake8: noqa
+from django.utils import timezone
 
 from api.preprint_providers.serializers import DeprecatedPreprintProviderSerializer
-from osf_tests.factories import PreprintProviderFactory
+from api.preprints.serializers import PreprintSerializer
+from osf_tests.factories import PreprintProviderFactory, PreprintFactory, NodeFactory
 from tests.base import DbTestCase
 from tests.utils import make_drf_request_with_version
 
 
+class TestRetractedPreprintSerialization(DbTestCase):
+
+    def setUp(self):
+        super(TestRetractedPreprintSerialization, self).setUp()
+        self.preprint = PreprintFactory()
+
+    def test_hidden_fields_on_retracted_preprint(self):
+        hide_if_withdrawal_fields = {
+             'is_published', 'is_preprint_orphan', 'license_record',
+             'tags', 'preprint_doi_created',
+        }
+        hide_if_not_withdrawal_fields = {'retraction_justification'}
+        always_show_fields = {
+            'date_created', 'date_modified', 'date_published', 'original_publication_date',
+            'doi', 'title', 'description', 'date_retracted'}
+
+        # test_non_retracted
+        req = make_drf_request_with_version()
+        result = PreprintSerializer(
+            self.preprint,
+            context={'request': req}).data
+        data = result['data']
+        attributes = set(data['attributes'])
+
+        assert_false(self.preprint.is_retracted)
```

Comment:
Not necessarily a blocker, but in general when writing new tests/test classes, prefer `pytest.fixture`s and plain `assert`s over `nose`-style `setUp`/`tearDown` and `assert_something`s.

Reply:
Please also test the behavior of `ever_public`